src/directories.ts

```diff
@@ -0,0 +1,16 @@
+let fs = require('fs');
+
+var folders: Array<string> = ['Scripts', 'Scenes', 'Prefabs', 'Materials', 'Textures', 'Audio'];
```

Comment:
These defaults are good, but maybe make this a configuration setting so the user can customize?

See VSCode docs: https://code.visualstudio.com/docs/extensionAPI/extension-points#_contributesconfiguration